shielddash/core/static/css/site.styl

```diff
@@ -210,3 +232,42 @@ p {
     opacity: 0.5;
   }
 }
+
+.switch-wrapper {
+  align-self: center;
+  display: flex;
+  font-size: 12px;
+  justify-content: space-between;
+  line-height: 20px;
+
+  .switch {
+    background: rgba(0,0,0,0.50);
+    border-radius: 30px;
+    box-shadow: inset 0px 1px 5px 0px rgba(0,0,0,0.20);
```

Comment:
Super small nitpick. If we wanted to save a few bytes, we could use `0` instead of `0px` here and in the other `box-shadow` values.